package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "jest --runInBand",
```

Comment:
Can This `test` script run the test coverage?

Reply:
@irfiacre, from my research, the test coverage script is setup in the `.travis.yml` file, not the `package.json`.

See https://docs.codeclimate.com/docs/travis-ci-test-coverage for more information.

Reply:
I am talking about the local covarage, the one that allows you to view if your tests are passing 100% locally!

Reply:
@irfiacre I understand now... After doing some research, I have updated the test script in `package.json` to display coverage in the console (CLI).

See https://jestjs.io/docs/en/configuration#coveragereporters-arraystring--stringany